habitat-baselines/habitat_baselines/rl/multi_agent/multi_agent_access_mgr.py

```diff
@@ -78,21 +78,25 @@ def __init__(
             updater_cls = MultiUpdater
             storage_cls = MultiStorage
 
+        # TODO(xavi to andrew): why do we call these functions? It seems they
```

Comment:
What does this TODO mean?

Reply:
I should use a different name :). I use this to list questions about the code that come up.

Here `policy_cls` can only be SelfBatchedPolicy and MultiPolicy. I notice that you call these with a long range of arguments. But those functions don't reuse any of those ([see here](https://github.com/facebookresearch/habitat-lab/blob/SIRo/habitat-baselines/habitat_baselines/rl/multi_agent/pop_play_wrappers.py#L123)). 

Can we just remove all those arguments and just keep agent, n_agents?